src/sentry/tagstore/base.py

```diff
@@ -396,16 +396,14 @@ def update_group_tag_key_values_seen(self, project_id, group_ids):
         """
         raise NotImplementedError
 
-    def get_group_tag_keys_and_top_values(self, project_id, group_id, environment_id, user=None):
-        from sentry.api.serializers import serialize
+    def get_group_tag_keys_and_top_values(self, project_id, group_id, environment_id, keys=None, limit=10):
```

Comment:
Same question about the choice of hardcoded `limit`, should the caller pass in 10 instead or is there a special existing reason for 10?

Reply:
This limit is just moved from the existing hardcoded one in the deleted code  at https://github.com/getsentry/sentry/pull/9905/files#diff-49568181b9aa8c6ac606d4b0a3772a9dL409

However you're right in that moving it here does make it kinda confusing, as it only gets used to limit the values, not the keys. I'll try come up with somthing better